src/DependencyInjection/FOSElasticaExtension.php

```diff
@@ -365,36 +346,36 @@ private function loadTypes(array $types, ContainerBuilder $container, array &$in
             }
 
             if (isset($type['indexable_callback'])) {
-                $indexableCallbacks[sprintf('%s/%s', $indexName, $name)] = $this->buildCallback($type['indexable_callback'], $name);
+                $indexableCallbacks[$indexName] = $this->buildCallback($type['indexable_callback'], $indexName);
             }
 
-            if ($container->hasDefinition('fos_elastica.serializer_callback_prototype')) {
-                $typeSerializerId = sprintf('%s.serializer.callback', $typeId);
-                $typeSerializerDef = new ChildDefinition('fos_elastica.serializer_callback_prototype');
-
-                if (isset($type['serializer']['groups'])) {
-                    $typeSerializerDef->addMethodCall('setGroups', [$type['serializer']['groups']]);
-                }
-
-                if (isset($type['serializer']['serialize_null'])) {
-                    $typeSerializerDef->addMethodCall('setSerializeNull', [$type['serializer']['serialize_null']]);
-                }
-
-                if (isset($type['serializer']['version'])) {
-                    $typeSerializerDef->addMethodCall('setVersion', [$type['serializer']['version']]);
-                }
-
-                $typeDef->addMethodCall('setSerializer', [[new Reference($typeSerializerId), 'serialize']]);
-                $container->setDefinition($typeSerializerId, $typeSerializerDef);
-            }
+//            if ($container->hasDefinition('fos_elastica.serializer_callback_prototype')) {
```

Comment:
@XWB Do you know what was the purpose of this?

Reply:
Ok it's about serializer integration.

Reply:
I make it work again

Reply:
Great!